### PR TITLE
Update Fighting_manage.md

### PR DESCRIPTION
**Pull request by NickPilotti**
updated fighting management md file

### CODE REVIEW

File: management/Fighting_manage.md [hunk 1]
```diff
@@ -28,7 +28,15 @@
 5. 06/30 - 07/6
 	* Manager: Nicholas Pilotti (nsp29)
 	* Goals:
-		1. GOAL1
+		1. Fixing movement issues
+		2. Fix FPS panicking at 60 frames
+		3. Insert designed background into combat area
+		4. Establish a floor according to the background
+		5. Establish window bounds for fighter (fighter cannot exit the screen in any direction)
+		6. Physics: Introduce gravity and acceleration for character and Hazards
```
Comment (nfarnan) [inline management/Fighting_manage.md:36]: This was a goal for the previous week that seemed to be implemented with the falling hazards, was it not?

**Review by nfarnan — CHANGES_REQUESTED** [review]

**Comment by nfarnan** [comment]
Some clarification needed on last week's vs this week's goals.

**Comment by NickPilotti** [comment]
Last weeks was to get the hazard moving, all it does is fall infinitely once the game is started, the goal is to get it to accelerate, and hit the ground correctly. Potentially add more

**Comment by nfarnan** [comment]
If this is going to be a goal for the week, I'll need specifics of what progress you will aim to complete this week.

**Comment by NickPilotti** [comment]
Did you get the updated version

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Review by nfarnan — APPROVED** [review]
Great, this specificity works for me.

**Pull request merged**